README.md

```diff
@@ -18,11 +18,33 @@ jobs:
           version: 1.7.5
           workspace: prod
 
-      # -- as an example --
+      # workflow now configured for use with terraform
 
       - name: Terraform apply
         run: |
           cd ops/my-terraform-configuration
           terraform init
           terraform apply
 ```
+
+Optionally, the Action can configure the following to allow use of a private GitHub repository as a [Terraform Module source](https://developer.hashicorp.com/terraform/language/modules/sources#github):
```

Comment:
Hey @magnetikonline, I'd like to see a little more detail on *when* this applies 😄